src/card.tsx

```diff
@@ -723,3 +729,199 @@ export const DisplayDataCard = React.forwardRef<HTMLDivElement, DisplayDataCardP
+                                    : undefined,
+                                paddingTop: withGradient && !icon && !hasTopActions ? 0 : 24,
+                            }}
+                        >
+                            {icon ? (
+                                <Box paddingBottom={withGradient ? 0 : 40} paddingX={24}>
+                                    {icon}
+                                </Box>
+                            ) : (
+                                <Box
+                                    paddingBottom={actions?.length || onClose ? (withGradient ? 24 : 64) : 0}
+                                />
+                            )}
+                            <Box
+                                paddingX={24}
+                                paddingTop={withGradient ? 40 : 0}
+                                paddingBottom={24}
+                                className={withGradient ? styles.displayCardGradient : undefined}
+                            >
+                                <Stack space={24}>
+                                    <div>
+                                        <Stack space={8}>
+                                            {(headline || pretitle || title) && (
+                                                <header>
+                                                    <Stack space={16}>
+                                                        {headline}
+                                                        <Stack space={4}>
+                                                            {pretitle && (
+                                                                <Text2
+                                                                    forceMobileSizes
+                                                                    truncate={pretitleLinesMax}
+                                                                    as="div"
+                                                                    regular
+                                                                    textShadow={textShadow}
+                                                                >
+                                                                    {pretitle}
+                                                                </Text2>
+                                                            )}
+                                                            <Text
+                                                                desktopSize={20}
+                                                                mobileSize={18}
+                                                                mobileLineHeight="24px"
+                                                                desktopLineHeight="28px"
+                                                                truncate={titleLinesMax}
+                                                                weight={textPresets.posterTitle.weight}
```

Comment:
please, use `weight={textPresets.cardTitle.weight}` and avoid create another poster title, maybe was a specs problems from our side